src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Overview/ThorchainSaversOverview.tsx

```diff
@@ -241,6 +243,26 @@ export const ThorchainSaversOverview: React.FC<OverviewProps> = ({
       .length,
   )
 
+  const isRunepoolWithdrawUnlocked = useMemo(() => {
```

Comment:
Nitpick if you feel like it for a cleanup follow-up after this is released (no need to track it as an issue, just keep note of it at your discretion, not a big deal as this component isn't so big): would be nice to abstract all the runepool bits fully into its utils/hooks, so that added branching/logic doesn't make the current components too large - we've seen how bone-dense these can be with LP!

Reply:
Note taken